apps/client/src/widgets/report/components/ipwon/components/jilbyeong.tsx

```diff
@@ -0,0 +1,44 @@
+import { Alert } from '@bds/ui';
+
+import { ALERT } from '@widgets/report/constant/alert-content';
+
+import { Accordion } from '../../accordion/accordion';
+import Graph from '../../graph/graph';
+
+const jilbyeongData = {
+  displayName: '질병입원일당(1일이상)',
+  surgery: {
+    productCoverage: 1000,
+    averageCoverage: 500,
+  },
+};
+
+const Jilbyeong = () => {
+  const isZero = jilbyeongData.surgery.productCoverage == 0;
```

Comment:
```suggestion
  const hasCoverage = jilbyeongData.surgery.productCoverage == 0;
```
는 어떤가요? 더 의미가 담겨있는 것 같은데

Reply:
다 합치고 바꿔주세용 ㅎ

Reply:
넵 다 합치고 하겠습니다 !